config/config-api/src/main/java/com/thoughtworks/go/config/SecretParam.java

```diff
@@ -54,6 +55,10 @@ public boolean isUnresolved() {
         return !resolved;
     }
 
+    public String placeholder() {
+        return String.format(SECRET_PARAM_SYNTAX, secretConfigId, key);
```

Comment:
This is no different than `asString()`.

Reply:
Incorporated review comment, it was done that way because at that time asString() was not implemented.